packages/schema/package.json

```diff
@@ -34,7 +34,8 @@
   "scripts": {
     "build": "node ./scripts/generate-schema-json.js",
     "clean": "git checkout -- ./lib/appium-config.schema.json || true",
-    "test:smoke": "node ./index.js"
+    "test:smoke": "node ./index.js",
+    "test": "exit 0"
```

Comment:
if we're running a subset of the tests via `npm test --workspace` and this workspace is one of them, `npm` will fail without a `test` script here, so that's why it's here.